src/apis/fitness/v1.ts

```diff
@@ -1917,7 +1917,7 @@ export namespace fitness_v1 {
      * millis since epoch). If specified, the API returns sessions modified
      * since this time. The page token is ignored if either start or end time is
      * specified. If none of start time, end time, and the page token is
-     * specified, sessions modified in the last 7 days are returned.
+     * specified, sessions modified in the last 30 days are returned.
```

Comment:
interesting that this changed from 7 to 30 days; is this a breaking change potentially?